src/indexers/vault-harvests-indexer.ts

```diff
@@ -13,6 +14,9 @@ export async function updateVaultHarvests() {
     const vaults = await chain.vaults.all();
 
     for (const vault of vaults) {
+      if (vault.address !== TOKENS.BVECVX) {
```

Comment:
on bveCVX gets updateVaultHarvests now?

Reply:
no, forgot to push my fixes after self review. thanks!